core/src/main/kotlin/Java/JavaPsiDocumentationBuilder.kt

```diff
@@ -167,7 +167,9 @@ class JavaPsiDocumentationBuilder : JavaDocumentationBuilder {
                         element.isInternal())
 
     private fun skipElementBySuppressedFiles(element: Any): Boolean =
-            element is PsiElement && element.containingFile.virtualFile.path in passConfiguration.suppressedFiles
+        element is PsiElement &&
+            element.containingFile.virtualFile != null &&
+            File(element.containingFile.virtualFile.path).absoluteFile in options.suppressedFiles
```

Comment:
`File` lacks an import and `options` have been replaced with `passConfiguration`. Also the type of `suppressedFiles` changed from `List<File>` to `List<String>`

Reply:
Sorry about that, merging from our fork did not work very well!